docs/repositories/repository-dashboard.md

```diff
@@ -38,7 +38,7 @@ Each tab displays the following information for the corresponding metric:
 !!! note
     The coverage tab only displays a value if Codacy received coverage data for the most recent commit. This is because one commit can easily change the size or number of files on the repository, or even remove some of the files that had coverage information.
 
-The chart also displays the **trendline** based on the past behavior and the **quality standard**  defined on the repository [quality settings](../repositories-configure/adjusting-quality-settings.md).
+The chart also displays the **trendline** based on the past behavior and the **quality goal** defined on the repository [quality settings](../repositories-configure/adjusting-quality-settings.md).
```

Comment:
I noticed that we used to call the dashed line on the quality evolution chart the "quality standard". I'm changing this to "quality goal".

Reply:
Then we need to change in the dashboard legend as well 
![image](https://user-images.githubusercontent.com/23286236/140298114-14b317a0-b4d9-4b0d-94ac-a2692ed722b3.png)

Reply:
You're right @inesgomas! I can open a pull request on codacy-spa to change the label on the chart to **Quality goal**, if you and @RaquelHipolito agree of course. But I believe in the future it will be confusing to have this dashed line called "Quality standard".

Reply:
I am fine with creating that PR

Reply:
Here's the pull request for that small change:

https://github.com/codacy/codacy-spa/pull/681